--- codesign_xpc	2018-12-28 11:30:57.000000000 +0100
+++ codesign_xpc.new	2018-12-28 11:31:01.000000000 +0100
@@ -10,7 +10,7 @@
     return ('"' + argument + '"' if ' ' in argument else argument)
 
 def _codesign_service(identity, path, entitlements_path=None):
-    command = ["codesign", "-fs", identity, path] + ([] if entitlements_path is None else ["--entitlements", entitlements_path])
+    command = ["codesign", "-o", "runtime", "-fs", identity, path] + ([] if entitlements_path is None else ["--entitlements", entitlements_path])
     log_message(" ".join(map(sanitize, command)))
 
     process = subprocess.Popen(command, stdout=subprocess.PIPE)
